<!DOCTYPE html>

<html lang="en-US">

<head>
    <!-- Meta tags, CSS links, and other head content here -->
    <title>CRIS Database</title>
    <link rel="icon" type="image/png" href="/cris/images/logo_icon.png">
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta name="description" content="CRIS: Crosslinking-based RNA Interactomes and Structurome">
    <meta name="author" content="https://zhipenglulab.org/">
    <meta name="keywords" content="RNA, PARIS, XLRNA, SHARC">
    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css">
    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.11.3/font/bootstrap-icons.css">
    <link rel="stylesheet" href="style.css">
    <!-- JavaScript functions -->
    <script src="functions.js"></script>
    <script src="https://code.jquery.com/jquery-3.5.1.slim.min.js"></script>
    <script src="https://cdn.jsdelivr.net/npm/@popperjs/core@2.5.3/dist/umd/popper.min.js"></script>
    <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/js/bootstrap.min.js"></script>
</head>

<body>
    <div id="navigation-container"></div>
    <!-- Main Contents-->
    <main>
        <div class="section-color">
            <h1>Why RNA?</h1><hr>
            <h2>Molecules to Medicine Mentality</h2>
                <p>
                    RNA, or <i>Ribonucleic Acid</i>, stands as a pivotal biomolecule crucial to life. There is a growing recognition of its potential as a therapeutic target. Originally viewed as a mere messenger molecule for shuttling genetic information from DNA to proteins, recent studies have illuminated RNA's multifaceted roles. Beyond gene regulation and protein synthesis, RNA orchestrates critical functions in cell signaling and facilitates intricate intra- and intercellular interactions. This is underscored by the emergence of RNA-based drugs approved for clinical use and numerous candidates in clinical trials, transforming RNA from an "undruggable" target to a promising frontier in disease treatment.
                        <sup>
                            <a href="#reflist1-4">[1-4]</a>
                        </sup>
                </p>
                <p>
                    The "Molecules to Medicine" approach emphasizes translating fundamental biological insights into clinical applications. In RNA research, this mindset underscores the journey from understanding RNA structures and functions at a molecular level to harnessing this knowledge for therapeutic use. This holistic approach integrates interdisciplinary expertise, combining molecular biology, bioinformatics, and pharmaceutical sciences to innovate diagnostics, therapies, and personalized medicine strategies. In recent decades, RNA research has witnessed explosive growth fueled by technological advancements and interdisciplinary collaborations. This expansion encompasses the study of RNA structures, functions, interactions, and regulatory mechanisms across diverse biological contexts. The field's evolution has been propelled by innovative physical and chemical methodologies that enable precise characterization of RNA molecules, revealing intricate details crucial for understanding their roles in health and disease.
                        <sup><a href="#reflist5-6">[5-6]</a></sup>
                </p>
            <h2>RNA Structures</h2>
                <p>
                    RNA molecules adopt intricate three-dimensional structures crucial for their diverse biological functions. Elucidation of such structures is pivotal, as it provides insights on their functional capabilities, such as catalysis, molecular recognition, and gene regulation. Despite strides in understanding RNA's complexity, unraveling its structural and interactive properties within living cells remains a formidable challenge. Characterizing RNA's secondary and tertiary structures proves particularly daunting due to the dynamic nature and complexity of RNAs in vivo. 
                        <sup><a href="#reflist7-9">[7-9]</a></sup>
                </p>
            <h2>Physical Methods</h2>
                <p>
                    Physical methods like X-ray crystallography, nuclear magnetic resonance (NMR) spectroscopy, and cryo-electron microscopy (cryo-EM) have been instrumental in deciphering RNA's structural complexity at atomic resolution. For instance, X-ray crystallography involves crystallizing RNA molecules and bombarding them with X-rays to determine their atomic structure. NMR spectroscopy analyzes the magnetic properties of RNA atoms, providing insights into their spatial arrangement in solution. Cryo-EM captures high-resolution images of RNA molecules in their native states, offering structural details without the need for crystallization. Notably, all of these techniques remain limited by their respective technologies, never truly capturing the complexity of RNA <i>in vivo</i>.
                        <sup><a href="#reflist10-15">[10-15]</a></sup>
                </p>
            <h2>Chemical Methods</h2>
                <p>
                    The advent of probing and next-generation sequencing-based techniques has revolutionized RNA research in cellular environments. These methods combine enzymatic and chemical treatments with high-throughput sequencing to capture detailed RNA data. Take for example, selective 2'-hydroxyl acylation analyzed by primer extension (SHAPE), dimethyl sulfate (DMS) probing. SHAPE identifies flexible and structured regions within RNA molecules by reacting with accessible 2'-hydroxyl groups. DMS probing reveals RNA bases that are unpaired or in double-stranded regions. Though informative, they still fall short when it comes to directly identifying specific interactions whilst also often being limited in the scope of their application.
                        <sup><a href="#reflist16-17">[16-17]</a></sup>
                </p>
            <h2>Crosslink-Ligate Principle</h2>
                <p>
                    The crosslink-ligate principle is an innovative approach that utilizes crosslinking reagents to stabilize and capture the interactions between spatially proximate RNA molecules. This technique allows researchers to map complex RNA networks and structural motifs by ligating the crosslinked RNA fragments. Recent advancements, such as in PARIS2, have optimized photochemistry and enzymology, enabling more efficient analysis of RNA structures and interactions in both cellular and viral contexts. Similarly, SHARC employs chemical reversible crosslinking to measure RNA 3D distances and alternative conformations within cells. These advances, combined with tools like CRSSANT, which enables classification and clustering of RNA crosslink-ligation data to uncover complex structures and homodimers, have revolutionized the way researchers probe RNA-protein and RNA-RNA interactions. By integrating these crosslinking methodologies with high-throughput sequencing and mass spectrometry, scientists can now unravel the intricate networks that govern cellular functions and disease mechanisms.
                        <sup><a href="#reflist18-20">[18-20]</a></sup>
                </p>

            <div class="separator-line"></div>
            <h3>References</h3>
            <div style="font-size: 8px" class="publications">
                <ul>
                    <!-- 1 - 4 -->
                    <li><a id="reflist1-4" href="https://www.ncbi.nlm.nih.gov/books/NBK21054/">Alberts, B., Johnson, A., Lewis, J., Raff, M., Roberts, K., & Walter, P. (2002). <i>Molecular Biology of the Cell</i> (4th ed.). Garland Science.</a></li>
                    <li><a id="reflist1-4" href="https://www.nature.com/articles/nature10887">Guttman, M., & Rinn, J. L. (2012). <i>Modular regulatory principles of large non-coding RNAs.</i> Nature, 482(7385), 339-346.</a></li>
                    <li><a id="reflist1-4" href="https://www.nature.com/articles/nrg2673#citeas">Licatalosi, D., Darnell, R. (2010). <i>RNA processing and its regulation: global insights into biological networks.</i> Nat Rev Genet, 11, 75–87.</a></li>
                    <li><a id="reflist1-4" href="https://pubmed.ncbi.nlm.nih.gov/29617640/">Crooke, S.T., Witztum, J.L., Bennett, C.F., & Baker, B.F. (2018). <i>RNA-Targeted Therapeutics.</i> Cell Metab, 27(4), 714-739.</a></li>

                    <!-- 5 - 6 -->
                    <li><a id="reflist5-6" href="https://www.nature.com/articles/s41573-022-00521-4">Childs-Disney, J.L., Yang, X., Gibaut, Q.M.R., et al. (2022). <i>Targeting RNA structures with small molecules.</i> Nat Rev Drug Discov, 21, 736–762.</a></li>
                    <li><a id="reflist5-6" href="https://www.nature.com/articles/s41419-022-05075-2">Zhu, Y., Zhu, L., Wang, X., et al. (2022). <i>RNA-based therapeutics: an overview and prospectus.</i> Cell Death Dis, 13, 644.</a></li>

                    <!-- 7 - 9 -->
                    <li><a id="reflist7-9" href="https://www.cell.com/cell/fulltext/S0092-8674(16)30422-6">Lu, Z., et al. (2016). <i>RNA Duplex Map in Living Cells Reveals Higher-Order Transcriptome Structure.</i> Cell, 165(5), 1267-1279.</a></li>
                    <li><a id="reflist7-9" href="https://pubmed.ncbi.nlm.nih.gov/30510063/">Lu, Z., & Chang, H.Y. (2018). <i>The RNA Base-Pairing Problem and Base-Pairing Solutions.</i> Cold Spring Harb Perspect Biol, 10(12).</a></li>
                    <li><a id="reflist7-9" href="https://www.nature.com/articles/s41467-020-20040-3">Lu, Z., Guo, J.K., Wei, Y., et al. (2020). <i>Structural modularity of the XIST ribonucleoprotein complex.</i> Nat Commun, 11, 6163.</a></li>

                    <!-- 10 - 15 -->
                    <li><a id="reflist10-15" href="https://pubmed.ncbi.nlm.nih.gov/9641982/">Ferré-D'Amaré, A.R., Zhou, K., & Doudna, J.A. (1998). <i>A general module for RNA crystallization.</i> J Mol Biol, 279(3), 621-631.</a></li>
                    <li><a id="reflist10-15" href="https://pubmed.ncbi.nlm.nih.gov/36903357/">Jackson, R.W., Smathers, C.M., & Robart, A.R. (2023). <i>General Strategies for RNA X-ray Crystallography.</i> Molecules, 28(5), 2111.</a></li>
                    <li><a id="reflist10-15" href="https://www.nature.com/articles/s41467-020-19371-y">Shi, H., Rangadurai, A., Abou Assi, H., et al. (2020). <i>Rapid and accurate determination of atomistic RNA dynamic ensemble models using NMR and structure prediction.</i> Nat Commun, 11, 5531.</a></li>
                    <li><a id="reflist10-15" href="https://pubmed.ncbi.nlm.nih.gov/36746110/">Marušič, M., Toplishek, M., & Plavec, J. (2023). <i>NMR of RNA - Structure and interactions.</i> Curr Opin Struct Biol, 79, 102532.</a></li>
                    <li><a id="reflist10-15" href="https://www.nature.com/articles/s41392-022-00916-0">Ma, H., Jia, X., Zhang, K., et al. (2022). <i>Cryo-EM advances in RNA structure determination.</i> Sig Transduct Target Ther, 7, 58.</a></li>
                    <li><a id="reflist10-15" href="https://www.nature.com/articles/s41592-021-01132-4">Das, R. (2021). <i>RNA structure: a renaissance begins?</i> Nat Methods, 18, 439.</a></li>

                    <!-- 16 - 17 -->
                    <li><a id="reflist16-17" href="https://www.nature.com/articles/nchembio.1131">Spitale, R., Crisalli, P., Flynn, R., et al. (2013). <i>RNA SHAPE analysis in living cells.</i> Nat Chem Biol, 9, 18-20.</a></li>
                    <li><a id="reflist16-17" href="https://www.nature.com/articles/nmeth.4057">Zubradt, M., Gupta, P., Persad, S., et al. (2017). <i>DMS-MaPseq for genome-wide or targeted RNA structure probing in vivo.</i> Nat Methods, 14, 75-82.</a></li>

                    <!-- 18 - 20 -->
                    <li><a id="reflist18-20" href="https://www.nature.com/articles/s41467-021-22552-y">Zhang, M., Li, K., Bai, J., Velema, W.A., Yu, C., van Damme, R., Lee, W.H., Corpuz, M.L., Chen, J.F., & Lu, Z. (2021). <i>Optimized photochemistry and enzymology enable efficient analysis of RNA structures and interactions in cells and virus infections.</i> Nat Commun, 12, 2344.</a></li>
                    <li><a id="reflist18-20" href="https://www.nature.com/articles/s41467-022-28602-3">Van Damme, R., Li, K., Zhang, M., Bai, J., Lee, W.H., Yesselman, J.D., Lu, Z., & Velema, W.A. (2022). <i>Chemical Reversible Crosslinking Enables Measurement of RNA 3D Distances and Alternative Conformations in Cells.</i> Nat Commun, 13, 911.</a></li>
                    <li><a id="reflist18-20" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC9104705/">Zhang, M., Hwang, I.T., Li, K., Bai, J., Chen, J.F., Weissman, T., Zou, J.Y., & Lu, Z. (2022). <i>Classification and clustering of RNA crosslink-ligation data reveal complex structures and homodimers.</i> Genome Res, 32, 1-18.</a></li>
                </ul>
            </div>
        </div>
        <div class="dot-container"><div class="separator-dots">&#9679;&#9679;&#9679;&#9679;&#9679;&#9679;&#9679;</div></div>
    
        <div class="section-color">
            <h1>PARIS: Psoralen Analysis of RNA Interactions and Structures</h1><hr />
            <p>
                Talk about the original idea behind PARIS. Discuss the improvements made in the pipeline. 
            </p>
            <div class="separator-line"></div>
            <h1>SHARC: Spatial 2′-Hydroxyl Acylation Reversible Crosslinking</h1><hr />
            <p>
                Talk about the concept of 2'-OH acylation and how this analysis is slightly different.
                Discuss the nuances in looking at the exo-trimmed samples.
            </p>
        </div>
        <div class="dot-container"><div class="separator-dots">&#9679;&#9679;&#9679;&#9679;&#9679;&#9679;&#9679;</div></div>

    </main>
    <div id="footer-container"></div>
</body>
</html>