<!DOCTYPE html>

<html lang="en-US">

<head>
    <!-- Meta tags, CSS links, and other head content here -->
    <title>CRIS Database</title>
    <link rel="icon" type="image/x-icon" href="icons8-x-96.png">
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta name="description" content="CRIS: Crosslinking-based RNA Interactomes and Structurome">
    <meta name="author" content="https://zhipenglulab.org/">
    <meta name="keyword" content="RNA, PARIS, XLRNA, SHARC">
    <link rel="stylesheet" type="text/css" href="style.css">
    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.11.1/font/bootstrap-icons.css">
    <!-- JavaScript functions -->
    <script src="functions.js"></script>
    <script src="igv.min.js"></script>
</head>

<body>
    <!-- Header -->
    <header>
        <div class="title">
            <u>C</u>rosslinking-based <u>R</u>NA <u>I</u>nteractomes and <u>S</u>tructuromes
        </div>
    </header>

    <!-- Side menu -->
    <div id="SiteNav" class="sidenav">
        <!-- Return to home link -->
        <div class="nav-section link-hover">
            <a class="nav-section-header" href="https://whl-usc.github.io/cris/home.html"> Home</a>
        </div>
        <!-- Sub-section; studies -->
        <div class="nav-section link-hover">
            <div class="nav-section-header">Studies</div>
                <a href="https://whl-usc.github.io/cris/crssant"> CRSSANT</a>
                <a href="https://whl-usc.github.io/cris/hiclip"> HiClip</a>
                <a href="https://whl-usc.github.io/cris/paris2"> PARIS</a>
                <a href="https://whl-usc.github.io/cris/sharc"> SHARC</a>
                <!-- Add more datasets as needed -->
        </div>
        <!-- Sub-section; resources -->
        <div class="nav-section link-hover">
            <div class="nav-section-header">Linked Resources</div>
                <a href="https://bedtools.readthedocs.io/en/latest/index.html">BEDtools</a>
                <a href="#">Custom Tools</a>
                <a href="https://whl-usc.github.io/cris/igv" class="tools-link">Integrative Genomics Viewer</a>
                <a href="https://www.htslib.org/" class="tools-link">SAMtools</a>
                <a href="https://github.com/alexdobin/STAR" class="tools-link">STAR</a>
                <!-- Add more tools as needed -->
            </div>
        <!-- Section; other links -->
        <div class="nav-section link-hover">
            <div class="nav-section-header">Downloads</div>
                <a href="https://whl-usc.github.io/cris/downloads.html">Data</a>
                <a href="https://whl-usc.github.io/cris/publications.html">Papers</a>
        <!-- Add more links as needed -->
        </div>
    </div>

    <!-- Footer -->
    <footer>
        <div class="footer-block">
            <div class="footer-item link-hover">
                <h3><u>Github</u></h3>
                    <a href="https://github.com/zhipenglu/CRSSANT/">CRSSANT</a>
                    <a href="https://github.com/minjiezhang-usc/PARIS2">PARIS</a>
                    <a href="https://github.com/whl-usc/">rna2d3d</a>
            </div>
            <div class="footer-item link-hover">
                <h3><u>Contact Us</u></h3>
                <a href="https://zhipenglulab.org/">Lab Website</a>    
                <a href="mailto:zhipengl@usc.edu" method="post">zhipengl@usc.edu</a>
            </div>
            <div class="footer-item link-hover">
                <h3><u>Mailing Address</u></h3>
                <a href="https://maps.app.goo.gl/FqTFehk5fkJEUTnv5" class="address">
                USC Alfred E. Mann School of Pharmacy and Pharmaceutical Sciences<br>
                University of Southern California<br>
                1985 Zonal Avenue, Los Angeles, CA 90033<br>
                </a>
            </div>
            <div class="footer-item copyright">
                <p><i><br>Decoding and Targeting RNA Machines in Living Cells</i></p>
                <p>&copy;&nbsp;&nbsp;2024 Lu Lab</p>
            </div>
        </div>
    </footer>

    <!-- Main Contents-->
    <main>
        <div class="section-color">
            <h1>CRIS</h1>
                <h2>The central resource hub dedicated to RNA-crosslinking methodologies and research.</h2>
                <hr></hr>
                <p>
                    Introducing the Crosslinking-based RNA Interactomes and Structurome (CRIS) database, a comprehensive resource advancing RNA research. CRIS consolidates data
                    from multiple pipelines, offering a user-friendly platform for researchers with varying computational expertise. It integrates data from established pipelines 
                    like Cross-linked RNA Secondary Structure Analysis using Network Techniques (CRSSANT), ensuring consistent preprocessing for accurate analysis. CRIS 
                    facilitates analyses across datasets, uncovering patterns in RNA structures, gene expression, RNA-RNA interactions, and providing insights into RNA dynamics 
                    and conformations.
                </p>
                <p>
                    Key features include detailed guidelines for data processing workflows, access to raw and processed data, statistical information, visualization tools, and 
                    GitHub-linked scripts.
                </p>
                <div class="container">
                    <div class="grid-container">
                        <a class="grid-item grid-hover" href="https://pass">BACKGROUND</a>
                        <a class="grid-item grid-hover" href="https://pass">DATA</a>
                        <a class="grid-item grid-hover" href="https://pass">INTERACTIONS</a>
                        <a class="grid-item grid-hover" href="https://pass">STATISTICS</a>
                        <a class="grid-item grid-hover" href="https://pass">TOOLS</a>
                    </div>
                </div>
                <div class="container"><h2>... AN OVERVIEW FIGURE HERE ...</h2></div>
        </div>

        <!-- References -->
        <hr></hr>
        <div class="section-color">
            <h1>Select Publications</h1>
                <p>These are the most relevant publications to the CRIS database.</p>
                <div class="publications">
                    <h2><b>PARIS</b></h2>
                        <ul>
                            <a href="https://www.nature.com/articles/s41467-021-22552-y">
                                Optimized photochemistry and enzymology enable efficient analysis of RNA structures and interactions in cells and virus infections.
                                Zhang M, Li K, Bai J, Velema WA, Yu C, van Damme R, Lee WH, Corpuz ML, Chen JF, Lu Z* <i>Nature Communications, 12:2344 </i>(2021).
                            </a>
                            <a href="https://www.dropbox.com/scl/fi/dnplp4rb5669lwvs976f1/2021_PARIS2.pdf?rlkey=vnl2yyjgupphirtc8cqmais7p&dl=0">
                                <b>PDF<b>
                            </a>
                        </ul>        
                        <ul>
                            <a href="https://link.springer.com/protocol/10.1007/978-1-4939-7213-5_4">
                                PARIS: psoralen analysis of RNA interactions and structures with high throughput and resolution.
                                Lu Z, Gong J and Zhang QC <i>Methods in Molecular Biology: RNA Detection, 59-84 </i>(2018).
                            </a>
                            <a href="https://www.dropbox.com/s/0m97bb2jriwunzh/2018_method.pdf?dl=0">
                                <b>PDF<b>
                            </a>
                        </ul>
                        <ul>
                            <a href="https://www.cell.com/cell/fulltext/S0092-8674(16)30422-6">
                                RNA duplex map in living cells reveals higher order transcriptome structure. 
                                Lu Z, Zhang QC, Lee B, Flynn RA, Smith MA, Robinson JT, Davidovich C, Gooding AR, Goodrich KJ, Mattick JS, 
                                Mesirov JP, Cech TR, Chang HY <i>Cell, 165 (5): 1267-1279 </i>(2016).
                            </a>
                            <a href="https://www.dropbox.com/s/epy34g26hqa951n/2016_paris%2Bs.pdf?dl=0">
                                <b>PDF<b>
                            </a>
                        </ul>
                    <h2><b>SHARC</b></h2>
                        <ul>
                            <a class="text links-grey" href="https://www.nature.com/articles/s41467-022-28602-3">
                                Chemical Reversible Crosslinking Enables Measurement of RNA 3D Distances and Alternative Conformations in Cells. 
                                Van Damme R, Li K, Zhang M, Bai J, Lee WH, Yesselman JD, Lu Z* and Velema WA* <i>Nature Communications, 13:911 </i>(2022).
                                </a>
                            <a href="https://www.dropbox.com/s/p8cv35fasoh2n4l/2022_SHARC.pdf?dl=0">
                                <b>PDF<b>
                            </a>
                        </ul>
                    <h2><b>CRSSANT</b></h2>
                        <ul>
                            <a href="https://genome.cshlp.org/content/32/5/968.long">
                                Classification and clustering of RNA crosslink-ligation data reveal complex structures and homodimers. 
                                Zhang M, Hwang IT, Li K, Bai J, Chen JF, Weissman T, Zou JY and Lu Z* <i>Genome Research, 32: 1-18 </i>(2022).
                            </a>
                            <a href="https://www.dropbox.com/scl/fi/vk1349njqh129b58hao5w/2022_crssant.pdf?rlkey=kq5xnt7m27d13xkl09xtpuycj&dl=0">
                                <b>PDF<b>
                            </a>
                        </ul>
                    <h2><b>METHODS</b></h2>
                        <ul>
                            <a href="https://www.sciencedirect.com/science/article/pii/S0076687923000873">
                                Chemical crosslinking and ligation methods for in vivo analysis of RNA structures and interactions. 
                                Lee WH, Li K and Lu Z* <i>Methods in Enzymology, 691: 253-281</i>(2023).
                                </a>
                            <a href="https://www.dropbox.com/scl/fi/d6anzarmi6wq19uncnunf/2023_MIE.pdf?rlkey=suuhg6e82lz68td884nf6wi42&dl=0">
                                <b>PDF<b>
                            </a>
                        </ul>
                </div>
        </div>


        <!-- Break up contents to new pages after each of the 3-linebreaks-->

        <!-- Background information -->
        <hr></hr><hr></hr><hr></hr>
        <div class="section-color">
            <h1>Why RNA?</h1>
                <hr></hr>
                <h2>Molecules to Medicine Mentality</h2>
                <hr></hr>
                <h2>RNA Structures</h2>
                <hr></hr>
                <h2>Physical Methods</h2>
                <hr></hr>
                <h2>Chemical Methods</h2>
                <hr></hr>
                <h2>Crosslink-Ligate Principle</h2>
                <hr></hr>
        </div>

        <!-- Datasets/Studies -->
        <hr></hr><hr></hr><hr></hr>
        <div class="section-color">
            <h1>PARIS (2)</h1>
            <hr></hr><hr></hr><hr></hr>
            <h1>SHARC-exo</h1>
        </div>

        <!-- Tools we use -->
        <hr></hr><hr></hr><hr></hr>
        <div class="section-color">
            <h1>Bioinformatic Tools</h1>
                <hr></hr>
                <h2>Trimmomatic</h2>
                <p>Trimmomatic is first used to remove the 3'-end adapters.</p>
                <div class="container">
                    <div class="code-block code-section">
                        java -jar trimmomatic-0.36.jar SE -threads 16 -phred33 $fastq $Outprefix'_trim3.fastq' ILLUMINACLIP:$fasta:3:20:10 SLIDINGWINDOW:4:20 MINLEN:18
                    </div>
                </div>
            <hr></hr>
                <h2>STAR</h2>
                <p>The mapping of pre-processed data is standardized from PARIS- and SHARC-based sequencing. STAR and various scripts from the CRSSANT pipeline are used.</p>
                <div class="container">
                    <div class="code-block code-section">
                        /STAR-2.7.1a/bin/Linux_x86_64/STAR<br>
                        --runThreadN 8<br>
                        --runMode alignReads<br>
                        --genomeDir $StaridxPath<br>
                        --readFilesIn $Fastq<br>
                        --outFileNamePrefix $Outprefix'_1_'<br>
                        --genomeLoad NoSharedMemory<br>
                        --outReadsUnmapped Fastx<br>
                        --outFilterMultimapNmax 10<br>
                        --outFilterScoreMinOverLread 0<br>
                        --outSAMattributes All<br>
                        --outSAMtype BAM Unsorted SortedByCoordinate<br>
                        --alignIntronMin 1<br>
                        --scoreGap 0<br>
                        --scoreGapNoncan 0<br>
                        --scoreGapGCAG 0<br>
                        --scoreGapATAC 0<br>
                        --scoreGenomicLengthLog2scale -1<br>
                        --chimOutType WithinBAM HardClip<br>
                        --chimSegmentMin 5<br>
                        --chimJunctionOverhangMin 5<br>
                        --chimScoreJunctionNonGTAG 0<br>
                        --chimScoreDropMax 80<br>
                        --chimNonchimScoreDropMin 20
                    </div>
                </div>
        </div>

        <!-- RNA-RNA interaction Table -->
        <hr></hr><hr></hr><hr></hr>
        <div class="section-color">
            <h1>RNA-RNA Interaction Table</h1>
                <p>A dynamic/interactive table will go here, detailing all known RNA-RNA interactions and the files that contain such information.</p>
            <div class="container">
            </div>
        </div> 
           
        <!-- Data Download Table -->
        <hr></hr><hr></hr><hr></hr>
        <div class="section-color">
            <h1>Available Data</h1>
                <h2>Processed Data</h2>
                    <p>Our data is available in BAM format...</p>
                    <p>Table of links to all types of data - raw vs. intermediary</p>

                <h2>Raw Data</h2>
                    <p><b>Option 1.</b>&nbsp;Bulk download data from a whole project via NCBI GEO.</p>
                    <ul>sra_grab.sh enables simple download and extraction of data from a GEO accession number.</ul>
                        <div class="container">
                            <div class="code-section">
                                sra_grab.sh
                            </div>
                        </div>
                    <p><b>Option 2.</b>&nbsp;Inividual sample (direct) download through NCBI GEO.</p>
                    <ul>At the bottom of GEO Accession Display, select <i>SRA Run Selector</i>.</ul>
                    <ul>In the run table, select the SRR<i>XXXXX</i> file in the <i>Run</i> column that corresponds to the correct data.</ul>
                    <ul>Under the <i>FASTA/FASTQ</i> download tab, click to download directly.</ul>
                    <ul>To download to a server, copy the link that opens in your browser upon clicking the <i>FASTQ</i> link; use wget or curl.</ul>
                    <ul>
                        <i>WARNING: Always check to see if a study uses paired-end or single-end sequences. 
                        If downloading through Option 2, be sure to download both files of the pair.</i>
                    </ul>
        </div>
    </main>
</body>

</html>