<!DOCTYPE html>

<html lang="en-US">

<head>
    <!-- Meta tags, CSS links, and other head content here -->
    <title>CRIS Database</title>
    <link rel="icon" type="image/x-icon" href="icons8-x-96.png">
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta name="description" content="CRIS: Crosslinking-based RNA Interactomes and Structurome">
    <meta name="author" content="https://zhipenglulab.org/">
    <meta name="keywords" content="RNA, PARIS, XLRNA, SHARC">
    <link rel="stylesheet" type="text/css" href="style.css">
    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.11.3/font/bootstrap-icons.min.css">
    <!-- JavaScript functions -->
    <script src="functions.js"></script>
</head>

<body>
    <div id="header-container"></div>
    <div id="navigation-container"></div>
    <!-- Main Contents-->
    <main>
        <div class="section-color">
            <!-- Homepage Contents -->
            <h1>CRIS</h1>
            <h2>The central resource hub dedicated to RNA-crosslinking methodologies and research.</h2><hr />
                <p>
                    Introducing the Crosslinking-based RNA Interactomes and Structurome (CRIS) database, a comprehensive resource advancing RNA research.
                </p>
                <p>
                    CRIS consolidates data from multiple pipelines, offering a user-friendly platform for researchers with varying computational expertise. It integrates data from established pipelines like Cross-linked RNA Secondary Structure Analysis using Network Techniques (CRSSANT), Selective 2'-Hydroxyl Acylation analyzed by Primer Extension (SHAPE), and Cross-linking and Immunoprecipitation (CLIP), ensuring consistent processing for accurate analysis. CRIS facilitates analyses across datasets, uncovering patterns in RNA structures, gene expression, RNA-RNA interactions, and providing insights into RNA dynamics and conformations.
                </p>
                <p>
                    Key features include detailed guidelines for data processing workflows, access to raw and processed data, statistical information, visualization tools, and quick access to relevant GitHub-linked scripts.
                </p>
                <div class="container">
                    <div class="grid-container">
                        <a class="grid-item grid-hover" href="#">BACKGROUND</a>
                        <a class="grid-item grid-hover" href="#">PASS</a>
                    </div>
                </div>
                <div class="separator-line"></div>
            <h2>Pipeline overview figure, a graphic abstract.</h2>
                <br /><br /><br /><br /><br /><br /><br /><br /><br /><br />
                <div class="separator-line"></div>
            <h1>Frequently Asked Questions</h1><hr />

                <div class="separator-line"></div>
            <h1>Common Solutions for Problems</h1><hr />
                <p>
                    Some of the most common problems that occur during data processing can be resolved by...
                </p>
                <div class="separator-line"></div>
            
            <!-- References -->
            <h1>Select Publications</h1>
                <div class="publications">
                    <p>
                        In the section below, you will find a curated list of publications pertinent to the CRIS database. The literature contains details that may be of interest, although the general workflow is described here.
                    </p>
                    <h3><b>PARIS</b></h3>
                        <ul>
                            <li>
                                <a href="https://www.nature.com/articles/s41467-021-22552-y" target="_blank">Optimized photochemistry and enzymology enable efficient analysis of RNA structures and interactions in cells and virus infections. Zhang M, Li K, Bai J, Velema WA, Yu C, van Damme R, Lee WH, Corpuz ML, Chen JF, Lu Z <i>Nature Communications, 12:2344 </i>(2021).</a>
                                <a href="https://www.dropbox.com/scl/fi/dnplp4rb5669lwvs976f1/2021_PARIS2.pdf?rlkey=vnl2yyjgupphirtc8cqmais7p&dl=0" target="_blank"><b>PDF</b></a>
                            </li>
                            <li>
                                <a href="https://link.springer.com/protocol/10.1007/978-1-4939-7213-5_4" target="_blank">PARIS: psoralen analysis of RNA interactions and structures with high throughput and resolution. Lu Z, Gong J and Zhang QC <i>Methods in Molecular Biology: RNA Detection, 59-84 </i>(2018).</a>
                                <a href="https://www.dropbox.com/s/0m97bb2jriwunzh/2018_method.pdf?dl=0" target="_blank"><b>PDF</b></a>
                            </li>
                            <li>
                                <a href="https://www.cell.com/cell/fulltext/S0092-8674(16)30422-6" target="_blank">RNA duplex map in living cells reveals higher order transcriptome structure. Lu Z, Zhang QC, Lee B, Flynn RA, Smith MA, Robinson JT, Davidovich C, Gooding AR, Goodrich KJ, Mattick JS, Mesirov JP, Cech TR, Chang HY <i>Cell, 165 (5): 1267-1279 </i>(2016).</a>
                                <a href="https://www.dropbox.com/s/epy34g26hqa951n/2016_paris%2Bs.pdf?dl=0" target="_blank"><b>PDF</b></a>
                            </li>
                        </ul>
                    <h3><b>SHARC</b></h3>
                        <ul>
                            <li>
                                <a class="text links-grey" href="https://www.nature.com/articles/s41467-022-28602-3" target="_blank">Chemical Reversible Crosslinking Enables Measurement of RNA 3D Distances and Alternative Conformations in Cells. Van Damme R, Li K, Zhang M, Bai J, Lee WH, Yesselman JD, Lu Z and Velema WA <i>Nature Communications, 13:911 </i>(2022).</a>
                                <a href="https://www.dropbox.com/s/p8cv35fasoh2n4l/2022_SHARC.pdf?dl=0" target="_blank"><b>PDF</b></a>
                            </li>
                        </ul>
                    <h3><b>CRSSANT</b></h3>
                        <ul>
                            <li>
                                <a href="https://genome.cshlp.org/content/32/5/968.long" target="_blank">Classification and clustering of RNA crosslink-ligation data reveal complex structures and homodimers. Zhang M, Hwang IT, Li K, Bai J, Chen JF, Weissman T, Zou JY and Lu Z <i>Genome Research, 32: 1-18 </i>(2022).</a>
                                <a href="https://www.dropbox.com/scl/fi/vk1349njqh129b58hao5w/2022_crssant.pdf?rlkey=kq5xnt7m27d13xkl09xtpuycj&dl=0" target="_blank"><b>PDF</b></a>
                            </li>
                        </ul>
                    <h3><b>METHODS</b></h3>
                        <ul>
                            <li>
                                <a href="https://www.sciencedirect.com/science/article/pii/S0076687923000873" target="_blank">Chemical crosslinking and ligation methods for in vivo analysis of RNA structures and interactions. Lee WH, Li K and Lu Z <i>Methods in Enzymology, 691: 253-281</i>(2023).</a>
                                <a href="https://www.dropbox.com/scl/fi/d6anzarmi6wq19uncnunf/2023_MIE.pdf?rlkey=suuhg6e82lz68td884nf6wi42&dl=0" target="_blank"><b>PDF</b></a>
                            </li>
                        </ul>
                </div>
        </div>

        <!-- Break up contents to new pages after each of the 3-line breaks-->

        <!-- RNA-RNA interaction Table -->
        <div class="dot-container"><div class="separator-dots">&#9679;&#9679;&#9679;&#9679;&#9679;&#9679;&#9679;</div></div>
        <div class="section-color">
            <h1>RNA-RNA Interactions</h1>
                <p>
                    A figure will go here, detailing all known RNA-RNA interactions (for instance, the snoRNA - tRNAs as two chromosomes next to each other).
                </p>
                <p>
                    Includes instructions on how to achieve the same style of figure.
                </p>
            <div class="container">
                <p>
                    FIGURE AND EXAMPLE
                </p>
            </div>
        </div>        
    </main>
    <div id="footer-container"></div>
</body>
</html>