<!DOCTYPE html>

<html lang="en-US">

<head>
    <!-- Meta tags, CSS links, and other head content here -->
    <title>CRIS Database</title>
    <link rel="icon" type="image/x-icon" href="icons8-x-96.png">
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta name="description" content="CRIS: Crosslinking-based RNA Interactomes and Structurome">
    <meta name="author" content="https://zhipenglulab.org/">
    <meta name="keywords" content="RNA, PARIS, XLRNA, SHARC">
    <link rel="stylesheet" type="text/css" href="style.css">
    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.11.1/font/bootstrap-icons.css">
    <!-- JavaScript functions -->
    <script src="include.js"></script>
</head>

<body>
    <div data-include-html="navigation.html"></div>
    <!-- Main Contents-->
    <main>
        <div class="section-color">
            <h1>CRIS</h1>
            <h2>The central resource hub dedicated to RNA-crosslinking methodologies and research.</h2>
            <hr />
            <p>
                Introducing the Crosslinking-based RNA Interactomes and Structurome (CRIS) database, a comprehensive resource advancing RNA research. CRIS consolidates data from multiple pipelines, offering a user-friendly platform for researchers with varying computational expertise. It integrates data from established pipelines like Cross-linked RNA Secondary Structure Analysis using Network Techniques (CRSSANT), ensuring consistent preprocessing for accurate analysis. CRIS facilitates analyses across datasets, uncovering patterns in RNA structures, gene expression, RNA-RNA interactions, and providing insights into RNA dynamics and conformations.
            </p>
            <p>
                Key features include detailed guidelines for data processing workflows, access to raw and processed data, statistical information, visualization tools, and GitHub-linked scripts.
            </p>
            <div class="container">
                <div class="grid-container">
                    <a class="grid-item grid-hover" href="https://pass">BACKGROUND</a>
                    <a class="grid-item grid-hover" href="https://pass">DATA</a>
                    <a class="grid-item grid-hover" href="https://pass">INTERACTIONS</a>
                    <a class="grid-item grid-hover" href="https://pass">STATISTICS</a>
                    <a class="grid-item grid-hover" href="https://pass">TOOLS</a>
                </div>
            </div>
            <div class="container">
                <h2>... AN OVERVIEW FIGURE HERE ...</h2>
            </div>
        </div>

        <!-- References -->
        <hr />
        <div class="section-color">
            <h1>Select Publications</h1>
            <p>
                In the section below are highlighted publications, relevant to the CRIS database.
            </p>
            <div class="publications">
                <h2><b>PARIS</b></h2>
                <ul>
                    <li>
                        <a href="https://www.nature.com/articles/s41467-021-22552-y" target="_blank">Optimized photochemistry and enzymology enable efficient analysis of RNA structures and interactions in cells and virus infections. Zhang M, Li K, Bai J, Velema WA, Yu C, van Damme R, Lee WH, Corpuz ML, Chen JF, Lu Z <i>Nature Communications, 12:2344 </i>(2021).</a>
                        <a href="https://www.dropbox.com/scl/fi/dnplp4rb5669lwvs976f1/2021_PARIS2.pdf?rlkey=vnl2yyjgupphirtc8cqmais7p&dl=0" target="_blank"><b>PDF</b></a>
                    </li>
                    <li>
                        <a href="https://link.springer.com/protocol/10.1007/978-1-4939-7213-5_4" target="_blank">PARIS: psoralen analysis of RNA interactions and structures with high throughput and resolution. Lu Z, Gong J and Zhang QC <i>Methods in Molecular Biology: RNA Detection, 59-84 </i>(2018).</a>
                        <a href="https://www.dropbox.com/s/0m97bb2jriwunzh/2018_method.pdf?dl=0" target="_blank"><b>PDF</b></a>
                    </li>
                    <li>
                        <a href="https://www.cell.com/cell/fulltext/S0092-8674(16)30422-6" target="_blank">RNA duplex map in living cells reveals higher order transcriptome structure. Lu Z, Zhang QC, Lee B, Flynn RA, Smith MA, Robinson JT, Davidovich C, Gooding AR, Goodrich KJ, Mattick JS, Mesirov JP, Cech TR, Chang HY <i>Cell, 165 (5): 1267-1279 </i>(2016).</a>
                        <a href="https://www.dropbox.com/s/epy34g26hqa951n/2016_paris%2Bs.pdf?dl=0" target="_blank"><b>PDF</b></a>
                    </li>
                </ul>

                <h2><b>SHARC</b></h2>
                <ul>
                    <li>
                        <a class="text links-grey" href="https://www.nature.com/articles/s41467-022-28602-3" target="_blank">Chemical Reversible Crosslinking Enables Measurement of RNA 3D Distances and Alternative Conformations in Cells. Van Damme R, Li K, Zhang M, Bai J, Lee WH, Yesselman JD, Lu Z and Velema WA <i>Nature Communications, 13:911 </i>(2022).</a>
                        <a href="https://www.dropbox.com/s/p8cv35fasoh2n4l/2022_SHARC.pdf?dl=0" target="_blank"><b>PDF</b></a>
                    </li>
                </ul>

                <h2><b>CRSSANT</b></h2>
                <ul>
                    <li>
                        <a href="https://genome.cshlp.org/content/32/5/968.long" target="_blank">Classification and clustering of RNA crosslink-ligation data reveal complex structures and homodimers. Zhang M, Hwang IT, Li K, Bai J, Chen JF, Weissman T, Zou JY and Lu Z <i>Genome Research, 32: 1-18 </i>(2022).</a>
                        <a href="https://www.dropbox.com/scl/fi/vk1349njqh129b58hao5w/2022_crssant.pdf?rlkey=kq5xnt7m27d13xkl09xtpuycj&dl=0" target="_blank"><b>PDF</b></a>
                    </li>
                </ul>

                <h2><b>METHODS</b></h2>
                <ul>
                    <li>
                        <a href="https://www.sciencedirect.com/science/article/pii/S0076687923000873" target="_blank">Chemical crosslinking and ligation methods for in vivo analysis of RNA structures and interactions. Lee WH, Li K and Lu Z <i>Methods in Enzymology, 691: 253-281</i>(2023).</a>
                        <a href="https://www.dropbox.com/scl/fi/d6anzarmi6wq19uncnunf/2023_MIE.pdf?rlkey=suuhg6e82lz68td884nf6wi42&dl=0" target="_blank"><b>PDF</b></a>
                    </li>
                </ul>
            </div>
        </div>

        <!-- Break up contents to new pages after each of the 3-linebreaks-->

        <!-- Background information -->
        <hr />

        <div class="section-color">
            <h1>Why RNA?</h1><hr>
            <h2>Molecules to Medicine Mentality</h2>
                <p>
                    RNA, or <i>Ribonucleic Acid</i>, stands as a pivotal biomolecule crucial to life, with growing recognition of its potential as a therapeutic target. Originally viewed as a mere messenger molecule for shuttling genetic information from DNA to proteins, recent research has illuminated RNA's multifaceted roles. Beyond gene regulation and protein synthesis, RNA orchestrates critical functions in cell signaling and facilitates intricate intra- and intercellular interactions. This expanded role is underscored by the emergence of RNA-based drugs approved for clinical use and numerous candidates in clinical trials, transforming RNA from an "undruggable" target to a promising frontier in disease treatment.
                </p>
                <p>
                    The "Molecules to Medicine" approach emphasizes translating fundamental biological insights into clinical applications. In RNA research, this mindset underscores the journey from understanding RNA structures and functions at a molecular level to harnessing this knowledge for therapeutical use. This holistic approach integrates interdisciplinary expertise, combining molecular biology, bioinformatics, and pharmaceutical sciences to innovate diagnostics, therapies, and personalized medicine strategies. In recent decades, RNA research has witnessed explosive growth fueled by technological advancements and interdisciplinary collaborations. This expansion encompasses the study of RNA structures, functions, interactions, and regulatory mechanisms across diverse biological contexts. The field's evolution has been propelled by innovative physical and chemical methodologies that enable precise characterization of RNA molecules, revealing intricate details crucial for understanding their roles in health and disease.
                </p>
            <h2>RNA Structures</h2>
                <p>
                    RNA molecules adopt intricate three-dimensional structures crucial for their diverse biological functions. Elucidation of such structures is pivotal, as it provides insights on their functional capabilities, such as catalysis, molecular recognition, and gene regulation. Despite strides in understanding RNA's complexity, unraveling its structural and interactive properties within living cells remains a formidable challenge. Characterizing RNA's secondary and tertiary structures proves particularly daunting due to the dynamic nature and complexity of RNAs in vivo. 
                </p>
            <h2>Physical Methods</h2>
                <p>
                    Physical methods like X-ray crystallography, nuclear magnetic resonance (NMR) spectroscopy, and cryo-electron microscopy (cryo-EM) have been instrumental in deciphering RNA's structural complexity at atomic resolution. For instance, X-ray crystallography involves crystallizing RNA molecules and bombarding them with X-rays to determine their atomic structure. NMR spectroscopy analyzes the magnetic properties of RNA atoms, providing insights into their spatial arrangement in solution. Cryo-EM captures high-resolution images of RNA molecules in their native states, offering structural details without the need for crystallization. Notably, all of these techniques remain limited by their respective technologies, never truly capturing the complexity of RNA <i>in vivo</i>.
                </p>
            <h2>Chemical Methods</h2>
                <p>
                    The advent of probing and next-generation sequencing-based techniques has revolutionized RNA research in cellular environments. These methods combine enzymatic and chemical treatments with high-throughput sequencing to capture detailed RNA data. Take for example, selective 2'-hydroxyl acylation analyzed by primer extension (SHAPE), dimethyl sulfate (DMS) probing. SHAPE identifies flexible and structured regions within RNA molecules by reacting with accessible 2'-hydroxyl groups. DMS probing reveals RNA bases that are unpaired or in double-stranded regions. Though informative, they still fall short when it comes to directly identifying specific interactions whilst also often being limited in the scope of their application.
                </p>
            <h2>Crosslink-Ligate Principle</h2>
                <p>
                    The crosslink-ligate principle  is an approach that involves crosslinking spatially proximate RNA molecules with specific reagents to stabilize and directly capture their interactions. Subsequent ligation of crosslinked RNA fragments allows the reconstruction of RNA networks and structural motifs. By integrating crosslinking with high-throughput sequencing or mass spectrometry, researchers can comprehensively map RNA-protein and RNA-RNA interactions, unraveling the intricate networks that underpin cellular functions.
                </p>
           <h3>References</h3>
                <div class="publications">
                    <ul>
                        <li>Lorem Ipsum</li>
                        <li>Lorem Ipsum</li>
                        <li>Lorem Ipsum</li>
                    </ul>
                </div>
        </div>

        <!-- Datasets/Studies -->
        <div class="section-color">
            <h1>PARIS (2)</h1>
            <hr></hr><hr></hr><hr></hr>
            <h1>SHARC-exo</h1>
        </div>

        <!-- Tools we use -->
        <div class="section-color">
            <h1>Bioinformatic Tools</h1>
            <hr />

            <h2>Data Pre-processing</h2>
                <p>
                    Sequencing reads are provided as compressed (zipped) fastq files. They must first be processed to remove sequencing adapters and unique barcodes, then demultiplexed using common tools. File prefixes and paths will vary according to the experimental conditions and file origin, and should be substituted for instances of <b>"$<i>Outprefix</i>"</b> or <b>"$<i>path</i>"</b> in subsequent steps.
                </p> 
            <h3>Trimmomatic</h3>
                <p>
                    When performing sequencing using an Illumina instrument, sequences that correspond to the library adapter can [and more often than not, are] present in the fastq files at the 5'- and 3'- ends of the reads if the read length is greater than the insert size. Since library inserts are shorter than the total read length of the sequencer, adapters must be trimmed away prior to alignment. Trimmomatic is used to remove the 3'-end adapters <a href="https://academic.oup.com/bioinformatics/article/30/15/2114/2390096" target="_blank"><i>(Bolger et al. 2014)</i></a>.
                </p>
                 <div class="container">
                    <code class="example">java -jar trimmomatic-0.36.jar SE -threads 16 -phred33 $fastq $Outprefix'_trim3.fastq' ILLUMINACLIP:$fasta:3:20:10 SLIDINGWINDOW:4:20 MINLEN:18</code>
                </div>
                <p> 
                    <i>Note that the described parameters are specific to this pipeline. For instance, the reads from PARIS and SHARC-based experiments are single-ended, therefore only one set of input and output files is specified.</i> The processing steps (trimming, cropping, adapter clipping, etc.) are specified as additional arguments after the input/output files. Multiple steps can be specified as required, by using additional arguments at the end as described in the section processing steps (separated by a single space). For input and output files, adding .gz/.bz2 as an extension tells Trimmomatic that the file is provided in gzip/bzip2 format or that Trimmomatic should gzip/bzip2 the file, respectively. Note that most processing steps take one or more settings, delimited by ':' (colon). 
                </p>
                <div class="container">
                    <code>java -jar &lt;trimmomatic.jar&gt; SE [-threads&lt;threads&gt;] [-phred33 | -phred64] &lt;input.fastq.gz&gt; &lt;output.fastq.gz&gt; ILLUMINACLIP:&lt;adapters.fa&gt;:&lt;seed mismatches&gt;:&lt;palindrome clip threshold&gt;:&lt;simple clip threshold&gt; SLIDINGWINDOW:&lt;windowSize&gt;:&lt;requiredQuality&gt; MINLEN:&lt;length&gt;</code>
                </div>
                <div class="dl-container">
                    <dl>
                        <dt>SE</dt>
                        <dd>Single ended mode.</dd>

                        <dt>-threads</dt>
                        <dd>Indicates the number of computing threads to use, which improves performance on multi-core computers. If not specified, it will be chosen automatically.</dd>

                        <dt>-phred33 or -phred64</dt>
                        <dd>Specifies the base quality encoding.</dd>

                        <dt>Input/output</dt>
                        <dd>For input and output files, specifying .gz tells Trimmomatic that the file provided is in compressed format or that Trimmomatic should gzip the file, respectively.</dd>

                        <dt>ILLUMINACLIP</dt>
                        <dd>Finds and removes specifically Illumina adapters.</dd>

                        <dt>fastaWithAdaptersEtc</dt>
                        <dd>Specifies the path to a fasta file containing all the adapters, PCR sequences, etc. The naming of the various sequences within this file determines how they are used.</dd>

                        <dt>seedMismatches</dt>
                        <dd>Specifies the maximum mismatch count which will still allow a full match to be performed. To speed up the search, short sections of each adapter (up to 16bp) are tested at all possible positions to find "seeds" that trigger a full alignment.</dd>

                        <dt>palindromeClipThreshold</dt>
                        <dd>Specifies how accurate the match between the two 'adapter ligated' reads must be for PE palindrome read alignment.</dd>

                        <dt>simpleClipThreshold</dt>
                        <dd>Specifies how accurate the match between any adapter etc. sequence must be against a read. Suggested values are 7-15 depending on the length of the Score; increase 0.6 per match, decrease by Q/10 per mismatch where Q is the quality score.</dd>

                        <dt>Sliding window</dt>
                        <dd>Performs sliding window trimming once the average quality within the window falls below specified threshold. By considering multiple bases, a single poor quality base will not cause the removal of high quality data later in the read.</dd>

                        <dt>Sliding window size</dt>
                        <dd>Length of window in number of base pairs.</dd>

                        <dt>Sliding window minimum quality</dt>
                        <dd>The average quality required during trimming.</dd>

                        <dt>Minimum read length</dt>
                        <dd>Removes reads that fall below the specified minimum length. Reads removed by this step are included in the 'dropped reads' count. Length written in base pairs.</dd>
                    </dl>
                </div>
                <p>
                    Note that this final pre-processing step takes place after the readCollapse and splitFastq (see below). Trimmomatic is used to remove the 5'-end adapter sequences. HEADCROP:17 option is used to trim off the designed random and multiplexing barcodes.
                </p>
                <div class="container">
                    <code class="example">java -jar trimmomatic-0.36.jar SE -threads 16 -phred33 $Outprefix'_trim3_nodup_'$barcode'.fastq' $Outprefix'_trim_no-dup_'$barcode'.fastq' HEADCROP:17 MINLEN:20</code>
                </div>
                <div class="container">
                    <code>java -jar &lt;trimmomatic.jar&gt; SE -threads 16 -phred33 &lt;input.fastq&gt; &lt;output.fastq&gt; HEADCROP:17 MINLEN:20</code>
                </div>

            <h3>icSHAPE Tools</h3>
                <p>
                    Carryover of code from the iCSHAPE pipeline. More details to be added.
                </p>
                <div class="container">
                    <code class="example">perl readCollapse $Outprefix'_trim3.fastq' $Outprefix'_trim3_nodup.fastq'</code>
                </div> 
                <div class="container">
                    <code>perl &lt;readCollapse.pl&gt; &lt;input.fastq&gt; &lt;output.fastq&gt;</code>
                </div>
                <div class="container">
                    <code class="example">perl splitFastq.pl -U $Outprefix'_trim3_nodup.fastq' -b 6:6 -l CGTGAT:R701::ACATCG:R702::GCCTAA:R703 -s -d ./</code>
                </div>
                <div class="container">
                    <code>perl splitFastq.pl -U &lt;input.fastq&gt; -b 6:6 -l L -s -d &lt;Output Path&gt;</code>
                </div>
                <div class="dl-container">
                    <dl>
                        <dt>-b</dt>
                        <dd>Specifies the position and length of hexamer for reads to be used for demultiplexing.</dd> 
                        <dt>-l&nbsp;&nbsp;“L”</dt>
                        <dd>Specifies the list of specific length (-b) of barcodes "L" to be used for separation and subsequent names for the output file(s). Multiple barcodes can be provided separated with two colons "::". For example, providing "CGTGAT:R701" will take reads with barcode "CGTGAT" and separate them into file "R701.fastq".</dd>
                        <dd>Use format BARCODE1:LIB_NAME1::BARCODE2:LIB_NAME2::...</dd>
                        <dt>-s</dt>
                        <dd>Generates a simple statistics output file if separated by library barcode.</dd>
                        <dt>-d</dt>
                        <dd>Specifies the directory at which to output file(s).</dd>

                    </dl>
                </div>
            <h2>Quality Control (FastQC)</h2>
            <h3>FastQC</h3>
                <p>
                    After pre-processing, data must be checked for quality.
                </p>
            <hr />
            <h2>Alignment to reference genome</h2>
            <h3>STAR</h3>
                <p>
                    The mapping of pre-processed data is standardized from PARIS- and SHARC-based sequencing. STAR and various scripts from the CRSSANT pipeline are used. More specific details can be found in the supplemental material provided as part of the <a href="https://genome.cshlp.org/content/suppl/2022/05/02/gr.275979.121.DC1/Supplemental_Material.pdf" target="_blank"><i>CRSSANT</i></a> pipeline paper.
                </p>
                <div class="container">
                    <code class="code-block example">
                        /STAR-2.7.1a/bin/Linux_x86_64/STAR<br />
                        --runThreadN 8<br />
                        --runMode alignReads<br />
                        --genomeDir $StaridxPath<br />
                        --readFilesIn $Fastq<br />
                        --outFileNamePrefix $Outprefix'_1_'<br />
                        --genomeLoad NoSharedMemory<br />
                        --outReadsUnmapped Fastx<br />
                        --outFilterMultimapNmax 10<br />
                        --outFilterScoreMinOverLread 0<br />
                        --outSAMattributes All<br />
                        --outSAMtype BAM Unsorted SortedByCoordinate<br />
                        --alignIntronMin 1<br />
                        --scoreGap 0<br />
                        --scoreGapNoncan 0<br />
                        --scoreGapGCAG 0<br />
                        --scoreGapATAC 0<br />
                        --scoreGenomicLengthLog2scale -1<br />
                        --chimOutType WithinBAM HardClip<br />
                        --chimSegmentMin 5<br />
                        --chimJunctionOverhangMin 5<br />
                        --chimScoreJunctionNonGTAG 0<br />
                        --chimScoreDropMax 80<br />
                        --chimNonchimScoreDropMin 20<br />
                    </code>
                </div>
            </div>

        <!-- RNA-RNA interaction Table -->
        <hr />
        <div class="section-color">
            <h1>RNA-RNA Interaction Table</h1>
                <p>A dynamic/interactive table will go here, detailing all known RNA-RNA interactions and the files that contain such information.</p>
            <div class="container">
            </div>
        </div> 
           
        <!-- Data Download Table -->
        <hr />
        <div class="section-color">
            <h1>Available Data</h1>
            <h2>Processed Data</h2>
                <p>
                    Our data is available in BAM format...
                </p>
                <p>
                    Table of links to all types of data - raw vs. intermediary
                </p>

            <h2>Raw Data</h2>
            <p>
                <b>Option 1.</b>&nbsp;Bulk download data from a whole project via NCBI GEO.
            </p>
            <p>
                sra_grab.sh enables simple download and extraction of data from a GEO accession number.
            </p>
            <p>
                <b>Option 2.</b>&nbsp;Individual sample (direct) download through NCBI GEO.
            </p>
            <div class="instructions">
                <ul>
                    <li>
                        At the bottom of GEO Accession Display, select <i>SRA Run Selector</i>.
                    </li>
                    <li>
                        In the run table, select the SRR<i>XXXXX</i> file in the <i>Run</i> column that corresponds to the correct data.
                    </li>
                    <li>
                        Under the <i>FASTA/FASTQ</i> download tab, click to download directly.
                    </li>
                    <li>
                        To download to a server, copy the link that opens in your browser upon clicking the <i>FASTQ</i> link; use wget or curl.
                    </li>
                </ul>
                <p><i>WARNING: Always check to see if a study uses paired-end or single-end sequences. If downloading through Option 2, be sure to download both files of the pair.</i></p>
            </div>
        </div>
    </main>

</body>

</html>