<!DOCTYPE html>
<html lang="en-US">
<head>
    <!-- Meta tags, CSS links, and other head content here -->
    <title>CRIS Database</title>
    <link rel="icon" type="image/png" href="/cris/images/logo_icon.png">
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta name="description" content="CRIS: Crosslinking-based RNA Interactomes and Structurome">
    <meta name="author" content="https://zhipenglulab.org/">
    <meta name="keywords" content="RNA, PARIS, XLRNA, SHARC">
    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css">
    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.11.3/font/bootstrap-icons.css">
    <link rel="stylesheet" href="style.css">
    <!-- JavaScript functions -->
    <script src="functions.js"></script>
    <script src="https://code.jquery.com/jquery-3.5.1.slim.min.js"></script>
    <script src="https://cdn.jsdelivr.net/npm/@popperjs/core@2.5.3/dist/umd/popper.min.js"></script>
    <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/js/bootstrap.min.js"></script>
</head>

<body>
    <div id="navigation-container"></div>
    <main>
        <div class="section-color">
            <h1>Bioinformatic Tools</h1><hr />
            <h2>Data Pre-processing</h2>
                <p>
                    Sequencing reads are provided as compressed (zipped) fastq files. They must first be processed to remove sequencing adapters and unique barcodes, then demultiplexed using common tools. File prefixes and paths will vary according to the experimental conditions and file origin, and should be substituted for instances of <b>"<i>$Outprefix</i>"</b> or <b>"<i>$path</i>"</b> in subsequent steps.
                </p> 
            <h3>Trimmomatic</h3>
                <p>
                    When performing sequencing using an Illumina instrument, sequences that correspond to the library adapter can be [more often than not] present in the fastq files at the 5'- and 3'- ends of the reads, if the read length is greater than the insert size. Since library inserts are shorter than the total read length of the sequencer, adapters must be trimmed away prior to alignment. Trimmomatic is used to remove the 3'-end adapters <a href="https://academic.oup.com/bioinformatics/article/30/15/2114/2390096" target="_blank"><i>(Bolger et al. 2014)</i></a>.
                </p>
                 <div class="container">
                    <code class="example">java -jar trimmomatic-0.36.jar SE -threads 16 -phred33 $fastq $Outprefix'_trim3.fastq' ILLUMINACLIP:$fasta:3:20:10 SLIDINGWINDOW:4:20 MINLEN:18</code>
                </div>
                <p> 
                    <i>Note that the described parameters are specific to this pipeline. For instance, the reads from PARIS and SHARC-based experiments are single-ended, therefore only one set of input and output files is specified.</i> The processing steps (trimming, cropping, adapter clipping, etc.) are specified as additional arguments after the input/output files. Multiple steps can be specified as required, by using additional arguments at the end as described in the section processing steps (separated by a single space). For input and output files, adding .gz/.bz2 as an extension tells Trimmomatic that the file is provided in gzip/bzip2 format or that Trimmomatic should gzip/bzip2 the file, respectively. Note that most processing steps take one or more settings, delimited by ':' (colon). 
                </p>
                <div class="container">
                    <code>java -jar &lt;trimmomatic.jar&gt; SE [-threads &lt;threads&gt;] [-phred33 | -phred64] &lt;input.fastq.gz&gt; &lt;output.fastq.gz&gt; ILLUMINACLIP:&lt;adapters.fa&gt;:&lt;seed mismatches&gt;:&lt;palindrome clip threshold&gt;:&lt;simple clip threshold&gt; SLIDINGWINDOW:&lt;windowSize&gt;:&lt;requiredQuality&gt; MINLEN:&lt;length&gt;</code>
                </div>
                <div class="dl-container">
                    <dl>
                        <dt>SE</dt>
                            <dd>Single ended mode.</dd>                
                        <dt>-threads</dt>
                            <dd>Indicates the number of computing threads to use, which improves performance on multi-core computers. If not specified, it will be chosen automatically.</dd>                
                        <dt>-phred33 or -phred64</dt>
                            <dd>Specifies the base quality encoding.</dd>            
                        <dt>Input/output</dt>
                            <dd>For input and output files, specifying .gz tells Trimmomatic that the file provided is in compressed format or that Trimmomatic should gzip the file, respectively.</dd>        
                        <dt>ILLUMINACLIP</dt>
                            <dd>Finds and removes specifically Illumina adapters.</dd>    
                        <dt>fastaWithAdaptersEtc</dt>
                            <dd>Specifies the path to a fasta file containing all the adapters, PCR sequences, etc. The naming of the various sequences within this file determines how they are used.</dd>
                        <dt>seedMismatches</dt>
                            <dd>Specifies the maximum mismatch count which will still allow a full match to be performed.</dd>
                            <dd>To speed up the search, short sections of each adapter (up to 16bp) are tested at all possible positions to find "seeds" that trigger a full alignment.</dd>
                        <dt>palindromeClipThreshold</dt>
                            <dd>Specifies how accurate the match between the two 'adapter ligated' reads must be for PE palindrome read alignment.</dd>
                        <dt>simpleClipThreshold</dt>
                            <dd>Specifies how accurate the match between any adapter etc. sequence must be against a read.</dd>
                            <dd>Suggested values are 7-15 depending on the length of the Score; increase 0.6 per match, decrease by Q/10 per mismatch where Q is the quality score.</dd>
                        <dt>Sliding window</dt>
                            <dd>Performs sliding window trimming when average quality within the window falls below specified threshold.</dd>
                            <dd>By considering multiple bases, a single poor quality base will not cause the removal of high quality data later in the read.</dd>
                        <dt>Sliding window size</dt>
                            <dd>Length of window in number of base pairs.</dd>
                        <dt>Sliding window minimum quality</dt>
                            <dd>The average quality required during trimming.</dd>
                        <dt>Minimum read length</dt>
                            <dd>Removes reads that fall below the specified minimum length. Reads removed by this step are included in the 'dropped reads' count. Length written in base pairs.</dd>
                    </dl>
                </div>
                <p>
                    <i>Note that this final pre-processing step takes place after readCollapse and splitFastq (see below).</i> Trimmomatic is used to remove the 5'-end adapter sequences. HEADCROP:17 option is used to trim off the designed random and multiplexing barcodes.
                </p>
                <div class="container">
                    <code class="example">java -jar trimmomatic-0.36.jar SE -threads 16 -phred33 $Outprefix'_trim3_nodup_'$barcode'.fastq' $Outprefix'_trim_no-dup_'$barcode'.fastq' HEADCROP:17 MINLEN:20</code>
                </div>
                <div class="container">
                    <code>java -jar &lt;trimmomatic.jar&gt; SE -threads 16 -phred33 &lt;input.fastq&gt; &lt;output.fastq&gt; HEADCROP:17 MINLEN:20</code>
                </div>
                <div class="separator-line"></div>
            <h3>icSHAPE Tools</h3>
                <p>
                    Carryover of code from the iCSHAPE pipeline. More details to be added.
                </p>
                <div class="container">
                    <code class="example">perl readCollapse $Outprefix'_trim3.fastq' $Outprefix'_trim3_nodup.fastq'</code>
                </div> 
                <div class="container">
                    <code>perl &lt;readCollapse.pl&gt; &lt;input.fastq&gt; &lt;output.fastq&gt;</code>
                </div>
                <div class="container">
                    <code class="example">perl splitFastq.pl -U $Outprefix'_trim3_nodup.fastq' -b 6:6 -l CGTGAT:R701::ACATCG:R702::GCCTAA:R703 -s -d ./</code>
                </div>
                <div class="container">
                    <code>perl splitFastq.pl -U &lt;input.fastq&gt; -b 6:6 -l L -s -d &lt;Output Path&gt;</code>
                </div>
                <div class="dl-container">
                    <dl>
                        <dt>-b</dt>
                            <dd>Specifies the position and length of hexamer for reads to be used for demultiplexing.</dd>                     
                        <dt>-l&nbsp;&nbsp;“L”</dt>
                            <dd>Specifies the list of specific length (-b) of barcodes "L" to be used for separation and subsequent names for the output file(s).</dd>
                            <dd>Multiple barcodes can be provided separated with two colons "::".</dd>
                            <dd>For example, providing "CGTGAT:R701" will take reads with barcode "CGTGAT" and separate them into file "R701.fastq".</dd>
                            <dd>Use format BARCODE1:LIB_NAME1::BARCODE2:LIB_NAME2::...</dd>                    
                        <dt>-s</dt>
                            <dd>Generates a simple statistics output file if separated by library barcode.</dd>                
                        <dt>-d</dt>
                            <dd>Specifies the directory at which to output file(s).</dd>
                    </dl>
                </div>
                <div class="separator-line"></div>
            <h2>Quality Control (FastQC)</h2><hr />
            <h3>FastQC</h3>
                <p>
                    After pre-processing, data must be checked for quality.
                </p>
                <div class="separator-line"></div>
            <h2>Alignment to reference genome</h2><hr />
            <h3>STAR</h3>
                <p>
                    The mapping of pre-processed data is standardized from PARIS- and SHARC-based sequencing. STAR and various scripts from the CRSSANT pipeline are used. More specific details can be found in the supplemental material provided as part of the <a href="https://genome.cshlp.org/content/suppl/2022/05/02/gr.275979.121.DC1/Supplemental_Material.pdf" target="_blank"><i>CRSSANT</i></a> pipeline paper.
                </p>
                <div class="container">
                    <code class="code-block example">
                        /STAR-2.7.1a/bin/Linux_x86_64/STAR<br />
                        --runThreadN 8<br />
                        --runMode alignReads<br />
                        --genomeDir $StaridxPath<br />
                        --readFilesIn $Fastq<br />
                        --outFileNamePrefix $Outprefix'_1_'<br />
                        --genomeLoad NoSharedMemory<br />
                        --outReadsUnmapped Fastx<br />
                        --outFilterMultimapNmax 10<br />
                        --outFilterScoreMinOverLread 0<br />
                        --outSAMattributes All<br />
                        --outSAMtype BAM Unsorted SortedByCoordinate<br />
                        --alignIntronMin 1<br />
                        --scoreGap 0<br />
                        --scoreGapNoncan 0<br />
                        --scoreGapGCAG 0<br />
                        --scoreGapATAC 0<br />
                        --scoreGenomicLengthLog2scale -1<br />
                        --chimOutType WithinBAM HardClip<br />
                        --chimSegmentMin 5<br />
                        --chimJunctionOverhangMin 5<br />
                        --chimScoreJunctionNonGTAG 0<br />
                        --chimScoreDropMax 80<br />
                        --chimNonchimScoreDropMin 20<br />
                    </code>
                </div>
            <h2>Primary Read Extraction and Classification</h2><hr />
            <h3>Extraction by samtools</h3>
                <div class="container">
                    <code class="code-block example">
                        samtools view -h x_Aligned.sortedByCoord.out.bam | awk '$1~/^@/ || NF<21' > x_nonchimeric_temp.sam<br />
                        samtools view -h x_Aligned.sortedByCoord.out.bam | awk '$1!~/^@/ && NF==21'> x_chimeric_temp.sam'<br />
                        samtools view -bS -F 0x900 -o x_nonchimeric_pri.bam x_nonchimeric_temp.sam<br />
                        samtools view -h x_nonchimeric_pri.bam > x_nonchimeric_pri.sam<br />
                        cat x_nonchimeric_pri.sam x_chimeric_temp.sam > x_1_pri.sam<br />
                        rm -f x_nonchimeric_temp.sam x_chimeric_temp.sam x_nonchimeric_pri.bam x_nonchimeric_pri.sam<br />
                    </code>
                </div>
            <h3>Classifying Primary Reads</h3>
                <div class="container">
                    <code class="code-block example">python3 gaptypes.py x_1_pri.sam x_1_pri -1 15 1</code>
                </div>
            <h3>Rearrange Softclipped Continuous Reads</h3>
                <div class="container">
                    <code class="code-block example">python softreverse_v2.py x_1_pricont.sam softrev.fastq</code>
                </div>
                <p>
                    Repeat STAR mapping with the softrev.fastq as input file, then extract and classify the alignments.
                </p>
            <h3>Combine output from the two rounds of STAR mapping</h3>
                <div class="container">
                    <code class="code-block example">
                        python3 merger_sams.py x_1_prigap1.sam x_2_prigap1.sam x gap1 x_prigap1.tmp<br />
                        python3 merger_sams.py x_1_prigapm.sam x_2_prigapm.sam x gapm x_prigapm.tmp<br />
                        python3 merger_sams.py x_1_prihomo.sam x_2_prihomo.sam x homo x_prihomo.tmp<br />
                        python3 merger_sams.py x_1_pritrans.sam x_2_pritrans.sam x trans x_pritrans.tmp<br /><br />
                        samtools view -H x_1_Aligned.sortedByCoord.out.bam > header<br /><br />
                        cat header x_prigap1.tmp > x_prigap1.sam<br />
                        cat header x_prigapm.tmp > x_prigapm.sam<br />
                        cat header x_prihomo.tmp > x_prihomo.sam<br /> 
                        cat header x_pritrans.tmp > x_pritrans.sam<br />
                        rm -f header *.tmp<br />
                    </code>
                </div>
            <h3>Filter splices and short gapped reads</h3>
                <div class="container">
                    <code class="code-block example">
                        python gapfilter.py Gtf x_prigap1.sam x_prigap1_filtered.sam 13 yes<br />
                        python gapfilter.py Gtf x_prigapm.sam x_prigapm_filtered.sam 13 yes<br />
                    </code>
                </div>
            <h2>Generate Alignment Statistics</h2><hr />
            <h3>Gap Length</h3>
                <div class="container">
                    <code class="code-block example">
                        cat x_prigap1_filtered.sam x_prigapm_filtered.sam > x_prigaps_filtered.sam<br/>
                        python gaplendist.py x_prigaps_filtered.sam sam x.list all<br/>
                        python gaplendist.py x.list list x_gaplen.pdf all<br/>
                        rm -f x_prigaps_filtered.sam x.list<br/>
                    </code>
                </div>
            <h3>Arm Length</h3>
                <div class="container">
                    <code class="code-block example">
                        cat x_prigap1_filtered.sam x_prigapm_filtered.sam x_pritrans.sam > x_prifiltered.sam<br />
                        python seglendist.py x_prifiltered.sam sam x_prifiltered.list<br />
                        python seglendist.py x_prifiltered.list list x_seglen.pdf<br />
                        rm -f x_prifiltered.sam x_prifiltered.list<br />
                    </code>
                </div>
            <h2>Read Counting</h2><hr />
                <div class="container">
                    <code class="code-block example">
                        samtools view -bS -o x_prigap1.bam x_prigap1.sam<br />
                        python CountCdsUtr.py x_prigap1.bam CdsUtr.bed none x_pri<br />
                    </code>
                </div>
                <div class="container">
                    <code class="code-block example">
                        bedtools bamtobed -i x_prigap1.bam > x.bed<br />
                        sort -k1,1 -k2,2n x.bed > x_sorted.bed<br />
                        rm x.bed<br />
                        bedtools coverage -g chrNameLength_sorted.txt -a reference.bed -b x_sorted.bed -counts -sorted -nonamecheck > x_coverage.txt<br />
                    </code>
                </div>
            <h2>CRSSANT</h2><hr />
                <div class="separator-line"></div>
            <h3>Input file preparation</h3>
                <div class="container">
                    <code class="code-block example">
                        python merger.py x_prigap1_filtered.sam x_pritrans.sam x_pri_crssant.sam<br />
                    </code>
                </div>
            <h3>OPTIONAL: Tag with unique identifier</h3>
                <p>
                    Prepare the files for CRSSANT DG/NG assembly, tagging with a label.
                </p>
                <div class="container">
                    <code class="code-block example">
                        awk '$0!~/^@/ {printf $1"-TAG1\t"; for(i=2;i<=NF;++i) printf $i "\t"; printf"\n"}' x_pri_crssant.sam > x_pri_crssant_TAG1_tmp.sam<br />
                        awk '$0!~/^@/ {printf $1"-TAG2\t"; for(i=2;i<=NF;++i) printf $i "\t"; printf"\n"}' x_pri_crssant.sam > x_pri_crssant_TAG2_tmp.sam<br />
                        samtools view -H x_pri_crssant.sam > header<br />
                        cat header x_pri_crssant_TAG1_tmp.sam x_pri_crssant_TAG2_tmp.sam > x_pri_crssant.sam<br />
                        rm -f header x_pri_crssant_TAG1_tmp.sam x_pri_crssant_TAG2_tmp.sam<br />
                    </code>
                </div>
                <p>
                    Separate the alignments by the tagged identifier.
                </p>
                <div class="container">
                    <code class="code-block example">
                        awk '$0~/^@/ || $1~/-TAG/' x_pri_crssant.cliques.t_o0.2.sam > x_ crssant_TAG1.sam
                    </code>
                </div>

            <h3>Build "genesfile" in BED format</h3>
                <div class="container">
                    <code class="code-block example">
                        python genes2bed.py x_ReadCount.txt CdsUtr.bed min_reads outname<br />
                    </code>
                </div>
            <h3>Generate bedgraphs</h3>
                <div class="container">
                    <code class="code-block example">
                        bedtools genomecov -bg -split -strand + -ibam x_pri_crssant.bam -g staridxPath/chrNameLength.txt > x_plus.bedgraph<br />
                        bedtools genomecov -bg -split -strand - -ibam x_pri_crssant.bam -g staridxPath/chrNameLength.txt > x_minus.bedgraph<br />
                    </code>
                </div>
            <h3>DG and NG assembly</h3><hr />
                <div class="container">
                    <code class="code-block example">
                        python crssant.py -cluster cliques -t_o 0.2 -out ./ x_pri_crssant.sam $Bed x_plus.bedgraph,x_minus.bedgraph<br />
                    </code>
                </div>
            <h3>TG assembly</h3><hr />
                <div class="container">
                    <code class="code-block example">
                        python gapmcluster.py x.bedpe x_gapm.sam
                    </code>
                </div>
            <div class="separator-line"></div>

            <h2>IGV Browser</h2><hr />
                <form id="uploadForm">
                    <label for="fileInput">Upload a file:</label>
                    <input type="file" id="fileInput" accept=".bam,.vcf,.bed,.gff,.gtf,.wig,.bigwig">
                </form>
                <div id="igv-container"></div>
                <script src="https://cdn.jsdelivr.net/npm/igv@2.15.13/dist/igv.min.js"></script>
        </div>
        <div class="dot-container"><div class="separator-dots">&#9679;&#9679;&#9679;&#9679;&#9679;&#9679;&#9679;</div></div>
    </main>
    <div id="footer-container"></div>
</body>
</html>